### PR TITLE
[10.0] account_cutoff_accrual_return

### PR DESCRIPTION
**Pull request by jbaudoux**
This PR depends on this other PR: ~~https://github.com/OCA/account-closing/pull/52~~ https://github.com/OCA/account-closing/pull/73

It allows to make accrual for returns to supplier, or from customer based on inventory on return location. It accrues the credit note to receive, or to send account.

### CODE REVIEW

**Comment by yvaucher** [comment]
Some PEP8 to fix:

```
account_cutoff_accrual_return/models/account_cutoff.py:24:80: E501 line too long (86 > 79 characters)
account_cutoff_accrual_return/models/account_cutoff.py:26:80: E501 line too long (86 > 79 characters)
account_cutoff_accrual_return/models/account_cutoff.py:31:28: F821 undefined name 'AccountCutOff'
account_cutoff_accrual_return/models/account_cutoff.py:59:9: F841 local variable 'company_currency_id' is assigned to 
```

**Review by ghost — COMMENTED** [review]
Fix Travis!

**Comment by jbaudoux** [comment]
@oleksandrpaziuk I fixed the pylint issue. Travis errors are for another unrelated module (multicurrency reevaluation)

**Comment by ghost** [comment]
to fix `multicurrency reevaluation` you need to update related module repo to the latest version

**Comment by ghost** [comment]
to fix flake8 correct this errors
```
======== Testing test_flake8 ========
account_cutoff_accrual_return/models/account_cutoff.py:23:80: E501 line too long (86 > 79 characters)
account_cutoff_accrual_return/models/account_cutoff.py:25:80: E501 line too long (86 > 79 characters)
account_cutoff_accrual_return/models/account_cutoff.py:56:9: F841 local variable 'company_currency_id' is assigned to but never used
```

**Comment by jbaudoux** [comment]
I don't see remaining issue in this PR. Regarding errors in `multicurrency reevaluation`module, the rebase didn't solve anything.

**Comment by ghost** [comment]
Red Travis is an issue
https://travis-ci.org/OCA/account-closing/builds/404456318?utm_source=github_status&utm_medium=notification

**Comment by jbaudoux** [comment]
@oleksandrpaziuk Travis will stay red until this PR https://github.com/OCA/account-closing/pull/77 is merged and other PR are rebased

**Comment by ghost** [comment]
@jbaudoux Yes. You are right!

**Comment by leemannd** [comment]
@jbaudoux It depends now on => https://github.com/OCA/account-closing/pull/73 and not the #52 
Can you update your PR description, so we are not lost?
The #77 has been merged :)

**Comment by sylvainvh** [comment]
@leemannd , it's green ! can you merge it ?

File: account_cutoff_accrual_return/README.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Account cutoff accrual returns
+==============================
+
+This module extends the functionality of account_cutoff_accrual_picking
+to allow the calculation of expense and revenue accruals on returned goods to supplier or from customer.
+
+The computation is done based on stock inventory on return locations marked as
+to be accrued.
+
+Installation
```
Comment (Tonow-c2c) [inline account_cutoff_accrual_return/README.rst:15]: Please Use ReadMe by fragment [https://github.com/OCA/maintainer-tools/tree/master/template/module/readme](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme)

**Comment by leemannd** [comment]
@sylvainvh I don't have rights on OCA repos. So I cannot do it.
There is one thing I know for sure, is that commits like:
- "fix after rebase"
- "flake8"
are not well considered and prone to change requests.
@jbaudoux Do you want to either:
- give me right of write on your fork of this repo, so I can update your PR?
- squash your commits so you keep only the meaningfull ones?

**Comment by pedrobaeza** [comment]
There's already another approach for the same problem (delivered or received goods that are not yet invoiced), that consists in post account entries for draft invoices, that are undo when validating the invoice. Maybe it's better to unify criteria for doing this task.

@sbidoul what do you think?

**Comment by sbidoul** [comment]
@pedrobaeza do you mean  `account_invoice_accrual`?

In theory account_invoice_accrual can do all your accruals, but I know some people do not like having draft invoices, so complementary, IMO.

**Comment by pedrobaeza** [comment]
Yeah, that's it, @sbidoul. I didn't remember now the exact name. I'm studying both options and this one is more complicated to be maintained IMO. That's why I ask @jbaudoux if they have considered the other flow.

**Comment by jbaudoux** [comment]
This PR is for goods you return to your supplier. Creating a draft refund could be idd another approach.

**Pull request closed without merging**